frontend/src/components/ProfileDropdown/ProfileDropdown.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../variables';
+
+.username {
+  margin: 0 0 5px 0;
+  text-align: right;
+  color: #e0e0e0;
```

Comment:
may i suggest switching to `hsl` (throughout the file)? a few other components and our vars seem to be using it